Dockerfile

```diff
@@ -1,24 +1,45 @@
-FROM node:6-alpine
-
-# Update latest available packages,
-# add 'app' user, and make temp directory
-RUN apk --no-cache add ffmpeg git && \
-    npm install -g grunt-cli bower && \
-    adduser -D app && \
-    mkdir /tmp/torrent-stream && \
-    chown app:app /tmp/torrent-stream
-
-WORKDIR /home/app
-COPY . .
-RUN chown app:app /home/app -R
-
-# run as user app from here on
-USER app
-RUN npm install && \
-    bower install && \
-    grunt build
-
-VOLUME [ "/tmp/torrent-stream" ]
+# Build peerflix server
+FROM node:14-alpine as build
+
+RUN apk --no-cache add git && \
+    npm install -g grunt bower 
+
+WORKDIR /app
+COPY package*.json ./
+
+COPY app app
+COPY server server
+COPY bower.json .
+COPY .bowerrc .
+COPY Gruntfile.js .
+COPY karma.conf.js .
+
+RUN npm install
+RUN bower --allow-root install
+RUN grunt build
+RUN npm prune --production
+
+RUN mkdir release && \
+    cp -r /app/node_modules release/node_modules && \
+    cp -r /app/dist release/app && \
```

Comment:
I'd rather differentiate between the two: `app` contains sources, whereas `dist` contains the optimized assets,
so maybe use `release/dist` instead?

Reply:
Fixed to release/dist